lldb/test/Shell/Recognizer/verbose_trap.test

```diff
@@ -1,4 +1,5 @@
 # UNSUPPORTED: system-windows
+# XFAIL: target=x86_64-{{.*}}-windows{{.*}}
```

Comment:
This shouldn't be necessary given the UNSUPPORTED clause above.

Reply:
And this test I'm pretty sure is only failing due to cmd line syntax issues, but no confirmation yet.